test/unit/style-spec/fixture/sources.output-api-supported.json

```diff
@@ -63,10 +76,24 @@
     "message": "source.coordinates: Unsupported property \"coordinates\"",
     "line": 22
   },
+  {
+    "message": "sources[4]: Source type must be vector, raster, or raster-dem",
+    "line": 20
+  },
+  {
+    "message": "sources[4]: Source url must be a valid Mapbox tileset url"
```

Comment:
Oh, probably because this fixture is totally missing the source `url`. Could we amend the fixture to _also_ include a source that has a `url` but that is an invalid value?

Reply:
> Could we amend the fixture to _also_ include a source that has a `url` but that is an invalid value?

We already have some fixtures like this – they just don't show up in the diff.

Reply:
ah, my mistake. thanks!